tests/install-deps.sh

```diff
@@ -10,9 +10,8 @@ if [ "${CI-}" == true ] ; then
 		# Semaphore installs more dependencies on their platform,
 		# they should be removed from here to save time.
 
-		# sudo apt-get update -qq
-		# sudo apt-get install -y $packagename
-		:
+		sudo apt-get update -qq
+		sudo apt-get install -y libseccomp-dev libseccomp2 # systemd deps
```

Comment:
I asked Semaphore to have those installed by default.

Meanwhile, can you use `sudo apt-get update -qq || true` as suggested in: https://github.com/coreos/rkt/pull/908#issuecomment-106003648